services/web-app/components/tabs/tabs.module.scss

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/motion' as motion;
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/theme' as theme;
+
+.tab-list {
+  display: flex;
+  max-width: min-content;
+  border-bottom: 1px solid theme.$border-subtle-01;
+  margin-right: -1rem;
+  margin-left: -1rem;
```

Comment:
```suggestion
  margin-right:  -(spacing.$spacing-05);
  margin-left: -(spacing.$spacing-05);
```

Reply:
the margin top from the website is not coming from the component, but from a parent element: 
<img width="1558" alt="image" src="https://user-images.githubusercontent.com/40550942/163875987-dfd3a691-f3e1-4e48-a6ae-d63f73a10ea4.png">

Do we want to add that padding directly at the component level on platform? I'm ok with that, just confirming

Reply:
I've been adding directly to the components, since we are wrapping them differently than Gatsby does. So I'd say yes, but we may have to look at spacing as a whole once we get real content into the site.

Reply:
👍 Added `spacing.$spacing-06` since that's the equivalent to 1.5rem which is what's being used in the site, lmk if we need to use `spacing.$spacing-07` and I'll change